packages/@aws-cdk/aws-stepfunctions-tasks/lib/codebuild/batch-delete-builds.ts

```diff
@@ -0,0 +1,58 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+import { generatePolicyResource } from './private/utils';
+
+/**
+ * Properties for CodeBuildBatchDeleteBuilds
+ */
+export interface CodeBuildBatchDeleteBuildsProps extends sfn.TaskStateBaseProps {
+  /**
+   * The IDs of the builds to delete.
+   */
+  readonly ids: string[];
```

Comment:
can this be a path reference to the state input? 
If it can, this will currently preclude the option to use `JsonPath.StringAt('$.Build..Arn')` as an input parameter and we should consider using the `TaskInput` (or a different) type instead

nit: might be better to call these `buildIds` to be specific and clear.
I do think it's implied so I'm not strongly opinionated here.